### PR TITLE
Refinements to the operator blog post

### PR DESCRIPTION
**Pull request by sdake**
Change ordered lists to unordered lists
Remove forward-looking statements
Edit for clarity

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Support
 
-Here are some resources to help you understand and use Istio:
+Test of checks Here are some resources to help you understand and use Istio:
```
Comment (geeknoid) [inline SUPPORT.md]: Remove.

Comment (sdake) [reply SUPPORT.md]: ya - not sure how that got in there.

File: content/en/blog/2019/introducing-istio-operator/index.md [hunk 2]
```diff
@@ -9,29 +9,28 @@ target_release: 1.4
...
-to configure the installation.
-The CRD is part of a new Istio operator implementation intended to simplify the common administrative tasks of
-installation, upgrade, and complex configuration changes for Istio.
-Validation and checking for installation and upgrade is tightly integrated with the tools to prevent
+a pattern for encoding human operational knowledge in software and are a popular way to simplify
+the administration of software infrastructure components. Istio is aa natural candidate for an automated
+operator as it is challenging to administer.
+
+[Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio. This approach
+worked well but had some disadvantages:
```
Comment (ostromart) [inline content/en/blog/2019/introducing-istio-operator/index.md]: I like @ssuchter 's proposal to invert the disadvantages to advantages:

[Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio. The new istioctl based installation has several advantages:

- Users just install one tool, istioctl
- All API fields are validated
- Small customizations not in the API do not require chart or API changes
- Version specific upgrade hooks can be easily and robustly implemented

Comment (sdake) [reply content/en/blog/2019/introducing-istio-operator/index.md]: addressed.

File: content/en/blog/2019/introducing-istio-operator/index.md [hunk 3]
```diff
@@ -9,29 +9,28 @@ target_release: 1.4
...
+[{{< istioctl >}} upgrade feature](/docs/setup/upgrade/istioctl-upgrade/).
+
+The `istioctl` operator commands use a
+[Custom Resource (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to configure the installation. The CRD is part of a new Istio operator
+implementation intended to simplify the common administrative tasks of installation, upgrade,
+and complex configuration changes for Istio. Validation and checking for installation and upgrade
+is tightly integrated with the tools to prevent
 common errors and simplify troubleshooting.
 
 ## Istio Control Plane API
```
[the remark this answers is not in the document]
Comment (sdake) [reply content/en/blog/2019/introducing-istio-operator/index.md]: addressed.

**Pull request merged**